recipes/default.rb

```diff
@@ -307,7 +315,8 @@
          :dela_ip => dela_ip,
          :nonconda_hosts_list => nonconda_hosts_list,
          :krb_ldap_auth => node['ldap']['enabled'].to_s == "true" || node['kerberos']['enabled'].to_s == "true",
-         :featurestore_jdbc_url => featurestore_jdbc_url
+         :featurestore_jdbc_url => featurestore_jdbc_url,
+         :registry_host => registry_host
```

Comment:
Do we need to store this in the variables at all? Can't we have Hopsworks itself looking it up with the consul client?

Reply:
Consul did not exist when I started on this work. Can we do it in a future Jira?